src/test/rgw/test_rgw_string.cc

```diff
@@ -12,6 +12,7 @@
  *
  */
 
+#include <errno.h>
```

Comment:
can you please move the include to rgw_string.h? i'm happy to merge once that builds